packages/http/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stoplight/prism-http",
-  "version": "5.8.2",
+  "version": "5.8.3",
```

Comment:
Is this required?  just check if it gets incremented bu build system during release or deployment ?

Reply:
Yes. we need to change the version as per the change.